templates/base.html

```diff
@@ -7,6 +7,9 @@
 <html xmlns:fb="http://www.facebook.com/2008/fbml" xmlns:og="http://ogp.me/ns#" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
 
     <head>
+        {% block extra_headers %}
+        {% endblock %}
```

Comment:
can we consider loading external js after the body, to prevent loading slowdowns?

Reply:
I am getting a js error if I place this block after.
Maybe it is possible to split the extra files in two sets, loading some in the header, and others at the end of the body.

Reply:
we should look further into this error. Do you have more details about what the problem is?